background.ts

```diff
@@ -214,7 +214,11 @@ Electron.app.on('before-quit', async(event) => {
 Electron.app.on('window-all-closed', () => {
   // On macOS, hide the dock icon.
   Electron.app.dock?.hide();
-  // On all platforms, we only quit via the notification tray / menu bar.
+  // On windows and macOS platforms, we only quit via the notification tray / menu bar.
+  // On Linux we close the application since not all distros support tray menu/icons
+  if (os.platform() === 'linux' && !settings.isFirstRun()) {
```

Comment:
Yes you are right, I noticed this issue yesterday eve. The point is that we need to distinguish somehow the first run otherwise this code gets executed once the user selects the default k8s version on first run as the app pops up a windows and then closes it. I will check if there is some other settings status that could be helpful here, I am not so sure about resetting isFirstRun, will think about it.